src/main/java/co/foxdev/foxbotng/api/PluginBase.java

```diff
@@ -0,0 +1,35 @@
+package co.foxdev.foxbotng.api;
+
+import co.foxdev.foxbotng.FoxBotNG;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Created by zack6849 on 11/29/2015.
+ */
+
+@Slf4j
+public abstract class PluginBase {
+
+    public abstract void onEnable();
+    public abstract void onDisable();
+
+    private Logger logger;
+
+    public Logger getLogger(){
```

Comment:
Can this be changed to use the logger that's already in the class from the @Slf4j annotation?